pkg/network/encoding/encoding_test.go

```diff
@@ -133,7 +133,7 @@ func getExpectedConnections(encodedWithQueryType bool, httpOutBlob []byte) *mode
 		},
 		AgentConfiguration: &model.AgentConfiguration{
 			NpmEnabled: false,
-			TsmEnabled: false,
+			UsmEnabled: false,
```

Comment:
I had to copy this from #15143 because of an incompatible change made in DataDog/agent-payload#223.